assets/js/navbar.js

```diff
@@ -20,13 +20,22 @@ const controllerNavbar = (function (jQuery) {
       const dropdownToggle = dropdownMenu.find('.dropdown-toggle');
       const dropdownMenuContent = dropdownMenu.find('.dropdown-menu');
 
-      dropdownToggle.on('mouseenter', function () {
-        dropdownMenuContent.addClass('show');
-      })
-
-      dropdownMenu.on('mouseleave', function () {
-        dropdownMenuContent.removeClass('show');
-      })
+      // Toggle dropdown menu when clicking the dropdown toggle
+      dropdownToggle.on('click', function (e) {
+        e.preventDefault(); // Prevent default anchor behavior
+        
+        // Close all other dropdowns
+        jQuery('.dropdown-menu').not(dropdownMenuContent).removeClass('show');
+        
+        dropdownMenuContent.toggleClass('show');
+      });
+
+      // Hide dropdown menu when clicking outside the dropdown menu
```

Comment:
It is a good practice to avoid comments that are unnecessary, the code should be well described and less comments the better. Because this means the code is clear ;)